upstairs/src/lib.rs

```diff
@@ -672,6 +695,10 @@ struct UpComs {
      * upstairs to all required downstairs has completed.
      */
     ds_done_tx: mpsc::Sender<u64>,
+    /**
+     * Notified when it's time to go active.
```

Comment:
Maybe say:  This channel is used to notify the proc task that it is time for this upstairs to go active.